hq/app/views/sgs/sgsAccount.scala.html

```diff
@@ -20,32 +22,48 @@ <h4 class="header light grey-text text-lighten-5">For @awsAccount.name</h4>
-            <p>The following Security Groups have ports open to the world, i.e. a CIDR set to <code>0.0.0.0/0</code>.</p>
-        </div>
 
-        @views.html.fragments.securityGroupHelp()
 
-        @if(hasSuppressedReports(flaggedSgs)) {
-            <div class="row">
-                <div class="card-panel valign-wrapper">
-                    <i class="material-icons left">visibility_off</i>
-                    <span class="sg-header__name">Some Security Groups have been ignored due to settings in AWS Trusted Advisor</span>
+        @flaggedSgsResult match {
+            case Right(flaggedSgs) => {
+                <div class="row">
+                    <h2>Open Security Groups</h2>
+                    <p>The following Security Groups have ports open to the world, i.e. a CIDR set to <code>0.0.0.0/0</code>.</p>
+                </div>
+
+                @views.html.fragments.securityGroupHelp()
+
+                @if(hasSuppressedReports(flaggedSgs)) {
+                <div class="row">
+                    <div class="card-panel valign-wrapper">
+                        <i class="material-icons left">visibility_off</i>
+                        <span class="sg-header__name">Some Security Groups have been ignored due to settings in AWS Trusted Advisor</span>
 
-                    <form class="sg-filter" action="#">
-                        <input class="js-sg-filter" type="checkbox" id="show-flagged-sgs" checked="checked" />
-                        <label class="black-text sg-filter__label" for="show-flagged-sgs">Show flagged</label>
-                        <input class="js-sg-filter" type="checkbox" id="show-ignored-sgs" />
-                        <label class="black-text sg-filter__label" for="show-ignored-sgs">Show ignored</label>
-                    </form>
+                        <form class="sg-filter" action="#">
+                            <input class="js-sg-filter" type="checkbox" id="show-flagged-sgs" checked="checked" />
+                            <label class="black-text sg-filter__label" for="show-flagged-sgs">Show flagged</label>
+                            <input class="js-sg-filter" type="checkbox" id="show-ignored-sgs" />
+                            <label class="black-text sg-filter__label" for="show-ignored-sgs">Show ignored</label>
+                        </form>
+                    </div>
                 </div>
-            </div>
-        }
+                }
 
-        <div class="row">
-            @views.html.fragments.openSecurityGroups(flaggedSgs, shadow=true)
-        </div>
+                <div class="row">
+                    @if(flaggedSgs.nonEmpty) {
+                        @views.html.fragments.openSecurityGroups(flaggedSgs, shadow=true)
+                    } else {
+                    <div class="card-panel valign-wrapper">
+                        <i class="material-icons medium green-text left">verified_user</i>
+                        <h5>No open Security Groups found</h5>
```

Comment:
Is `<h5>` part of the hierarchy here, or is it just chosen for its appearance?

Reply:
Chosen for appearance, to circumvent adding more css :)

If there is a preferred convention, happy to follow